src/types/new_types/securified_persona.rs

```diff
@@ -3,114 +3,99 @@ use crate::prelude::*;
 /// The `SecurifiedEntityControl`, address and possibly third party deposit state of some
 /// Securified entity.
 #[derive(Clone, Debug, PartialEq, Eq, Hash)]
-pub struct SecurifiedEntity {
+pub struct SecurifiedPersona {
+    name: String,
     /// The address which is verified to match the `veci`
-    address: AddressOfAccountOrPersona,
-
+    identity_address: IdentityAddress,
     securified_entity_control: SecurifiedEntityControl,
-
     /// If we found this UnsecurifiedEntity while scanning OnChain using
     /// Gateway, we might have been able to read out the third party deposit
     /// settings.
     third_party_deposit: Option<ThirdPartyDepositPreference>,
```

Comment:
We can remove this now that it is specific to Persona

Reply:
yeah wont be translated over to Sargon.